src/components/ToDoItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Button } from "./Button";
+import { Input } from "./Input";
+
+type ToDoItemProps = {
+  item: { id: number; text: string };
```

Comment:
Please, create the` type ToDoItem = {id: number; text: string };` and use it here and below